tests/unit_tests/bittensor_tests/test_subtensor.py

```diff
@@ -16,14 +16,14 @@ def setUp(self):
         self.wallet.set_hotkey(Keypair.create_from_mnemonic(Keypair.generate_mnemonic()), encrypt=False, overwrite=True)
         self.mock_neuron = self.subtensor._neuron_dict_to_namespace(
             dict({
-                "version":0,
+                "version":1,
```

Comment:
Why was this required?

Reply:
There was a check in neuron dict to namespace that if version is 0 then return a null neuron